### PR TITLE
Improve docs: liquidity.proto, query.proto, tx.proto, batch.go and a few other files

### PR DESCRIPTION
**Pull request by kogisin**
## Description

The `develop` branch that @barriebyron use on these PRs #399, #400, #401 are 135 commits behind. Merging with the latest `develop` branch doesn't seem to catch all the updates. I created this PR to make modification from those PRs since the changes are not that many.

closes: #177

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Review by barriebyron — APPROVED** [review]
it looks like my edits got in here, thanks @kogisin for getting the right develop branch in place. As we chatted about on Slack, I would prefer if we use one file per PR as the original PRs did. Also, moving forward let's not go moving files around between PRs. The owner of the original PR is responsible for resolving problems (oops that's me!). So instead of moving multiple files into a single PR as we did here, we'll do better in the future (we are all learning, that's a good thing!). Good intentions, but more overhead.

**Comment by kogisin** [comment]
My intention was to move forward fast since we're getting close for the liquidity module to be on Cosmos Hub at the end of this month. Apologize for moving things around in a hurry.  I should have waited until I get an approval from you. As we chatted on Slack, let's work in smaller chunk in the future.

**Review by dongsam — APPROVED** [review]

**Pull request merged**
